package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "music-app",
+  "description": "This is a small music react app",
+  "scripts": {
+    "format": "prettier --write \"src/**/*.{js,jsx}\"",
+    "lint": "eslint \"src/**/*.{js,jsx}\" --quiet",
```

Comment:
Why are you passing`--quiet` flag?

Reply:
To report errors only.

Reply:
Why you want to skip warnings?

Reply:
This part recommended by "React Videos",
He said warning are not big deals an errors are more important so better to focus on them.

but if warnings are necessary I will delete that part.

Reply:
That's ok for now. But sometimes warnings can be very beneficial. Thanks for the explanation 👍🏽